pkg/recipes/terraform/execute.go

```diff
@@ -61,6 +62,8 @@ type executor struct {
 
 	// k8sClientSet is the Kubernetes client.
 	k8sClientSet kubernetes.Interface
+
+	armOptions *arm.ClientOptions
```

Comment:
I don't this is used anymore in this terraform package?

Reply:
removed it